packages/editor/src/private-apis.js

```diff
@@ -15,6 +15,7 @@ import PostPanelRow from './components/post-panel-row';
 import PostViewLink from './components/post-view-link';
 import PreviewDropdown from './components/preview-dropdown';
 import PreferencesModal from './components/preferences-modal';
+import { useIsDirty } from './components/entities-saved-states/hooks/use-is-dirty';
```

Comment:
I think this is already exported as `useEntitiesSavedStatesIsDirty`, but honestly, I don't mind making this hook private.

@okmttdhr, do you know if it's used by any third-party code of WP/Gutenberg, as you made the hook public?